### PR TITLE
chore: update docs/tests to use aws-amplify version 5

### PR DESCRIPTION
**Pull request by esauerbo**
#### Description of changes

Updated install scripts for react-liveness and canary apps to use aws-amplify@5

#### Description of how you validated changes

Searched for instances of aws-amplify in repo and checked they reference v5
Ran/viewed docs page 
Screenshot 2023-10-17 at 11 24 17 PM

Ran build system tests as a pr hook against this branch
Screenshot 2023-10-11 at 1 01 45 PM
#### Checklist

- [ ] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
- [ ] PR description included
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] `yarn test` passes and tests are updated/added
- [ ] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by thaddmt — APPROVED** [review]
LGTM

File: docs/src/pages/[platform]/getting-started/migration/migration.react.mdx [hunk 1]
```diff
@@ -252,14 +252,14 @@ This export has been removed and should no longer be used.
 ### 1.x to 2.x
 #### Installation
 
-Install the 2.x version of the `@aws-amplify/ui-react` library.
+Install the 2.x version of the `@aws-amplify/ui-react` library and the 5.x version of the `aws-amplify` library.
```
Comment (reesscot) [inline docs/src/pages/[platform]/getting-started/migration/migration.react.mdx:255]: Not sure this is the case, did you check what our aws-amplify peerDependency was at this ui-react version?

File: docs/src/pages/[platform]/getting-started/migration/migration.vue.mdx [hunk 2]
```diff
@@ -51,16 +51,16 @@ If you were using `I18n` to translate those keys, please update your translation
 
 ## 1.x to 2.x
 ## Installation
-Install the 2.x version of the `@aws-amplify/ui-vue` library.
+Install the 2.x version of the `@aws-amplify/ui-vue` library and the 5.x version of the `aws-amplify` library.
```
Comment (reesscot) [inline docs/src/pages/[platform]/getting-started/migration/migration.vue.mdx:54]: Did you validate the peerDependency version of `aws-amplify` at ui-vue version 2?

File: docs/src/pages/[platform]/getting-started/migration/migration.react.mdx (hunk 1 above)
Comment (esauerbo) [reply docs/src/pages/[platform]/getting-started/migration/migration.react.mdx]: Yeah looks like it's 3 or 4 so removing the version from aws-amplify

**Pull request closed without merging**